go.mod

```diff
@@ -28,16 +28,19 @@ require (
 	github.com/docker/go-connections v0.4.0 // indirect
 	github.com/docker/go-units v0.4.0
 	github.com/ftrvxmtrx/fd v0.0.0-20150925145434-c6d800382fff
+	github.com/gogo/protobuf v1.3.2 // indirect
```

Comment:
What's pulling this in?

Reply:
My best guess is `github.com/Netflix/titus-kube-common` bump

We don't seem to be actually using it in the titus-executor codebase directly.
go mod tidy doesn't make it go away

Reply:
Oh, k8s itself?
```kubernetes/go.mod:	github.com/gogo/protobuf v1.3.2```